src/search.ts

```diff
@@ -36,6 +37,13 @@ export interface SearchResult {
     lastResultUrl?: string;
 }
 
+export interface SearchFilter {
+    key: string;
+    value: string[];
+    comparisonOperator: '~' | '==' | '!=' | '>' | '<' | '>=' | '<=';

```

Comment:
I think we can remove this operator `~`. @emilhdiaz, @rsmayda is there a use case you guys see in which we would need `~`?

Reply:
Yes, the use case where this operator was needed really is outside of the scope of an ES `filter` and covers instead the `must` use cases. See @rsmayda's comments in the sister MR: 

https://github.com/awslabs/fhir-works-on-aws-search-es/pull/30/files#r544635787
https://github.com/awslabs/fhir-works-on-aws-search-es/pull/30/files#r544636626

Removed from the interface and will update the search service.